### DISCUSSION TITLE
Does getx support FirebaseAnalyticsObserver?

### DISCUSSION THREAD

**Discussion by litopjos** [jonataslaw/getx#1946] [Q&A]
Does getx support FirebaseAnalyticsObserver to track routes visited using Firebase Analytics?

There was a previous post here that said yes but the accompanying code snippet was using MaterialApp to associate the FirebaseAnalyticsObserver instance to the MaterialApp:navigatorObservers property which confuses me cuz doesnt getx replace MaterialApp with GetMaterialApp which I dont think has a navigatorObservers property.

Thanks in advance for any assistance given.

/Jose

**Comment by jonataslaw** [comment]
<img width="474" alt="Captura de Tela 2021-10-27 às 21 42 39" src="https://user-images.githubusercontent.com/35742643/139167069-ce17b6cd-96f2-4a3c-a94a-be48b0e79f09.png">
